### ISSUE TITLE
NPM module

### ISSUE THREAD

**Issue by flochtililoch** [alexbain/lirc_web#20]
Would you consider publishing this as a standalone NPM module?

**Comment by alexbain** [comment]
Certainly would. If you'd be willing to take a stab at it I'd be happy to review, as well.

**Comment by alexbain** [comment]
@flochtililoch I did some thinking about this, and I'm not sure how easy it would be to turn this into a standalone NPM module. I've made choices about templating languages, css preprocessors, and build steps here that might not merge well with other projects.

One thought I had was if the API was extracted from `lirc_web` into an NPM module called something like `lirc_api`. This would allow anyone to embed just the API into an existing Express app. Would that be helpful to you?

**Comment by flochtililoch** [comment]
I was more picturing the use of this as a module installed globally, along with a [`bin` entry](https://docs.npmjs.com/files/package.json#bin) and [`preferGlobal`](https://docs.npmjs.com/files/package.json#preferglobal) in `package.son`. This would allow to run the app as a binary, and also enable easier install / update without relying on git dependency.

**Comment by alexbain** [comment]
Oh, I see what you mean. I hadn't considered that before, but I like the idea. If I adjusted the check for the `config.json` (https://github.com/alexbain/lirc_web/blob/master/app.js#L46) to check the current directory first, then the user's home directory for `.config.json`, it'd be a pretty seamless change.

**Comment by alexbain** [comment]
@flochtililoch Try this out, and let me know if this does what you expect:

```
sudo npm install -g git://github.com/alexbain/lirc_web.git#standalone-script
lirc_web
```

If you have a `config.json` file, move it to the home directory of your user and rename it to `.lirc_web_config.json` and it will be detected.

**Comment by flochtililoch** [comment]
@alexbain yep that's just what I need :+1:

**Comment by alexbain** [comment]
Okay, this is merged into master and tagged as `v0.2.1`.

You should now be able to do `npm install -g lirc_web` and it'll do what you expect. Great suggestion!

**Comment by flochtililoch** [comment]
awesome!
